bedrock/firefox/views.py

```diff
@@ -351,17 +341,17 @@ def show_default_account_whatsnew(version):
 
 
 class FirstrunView(L10nTemplateView):
-    ftl_files_map = {
-        "firefox/firstrun/firstrun.html": ["firefox/firstrun"],
-        "firefox/developer/firstrun.html": ["firefox/developer"],
-    }
+    ftl_files_map = {"firefox/developer/firstrun.html": ["firefox/developer"], "firefox/new/desktop/download.html": ["firefox/new/desktop"]}
 
     def get(self, *args, **kwargs):
         version = self.kwargs.get("version") or ""
+        new_page_url = reverse("firefox.new") + "?reason=outdated"
```

Comment:
Just a tad nicer way to do this is:
```python
>>> from bedrock.base.templatetags.helpers import urlparams
>>> urlparams(reverse("firefox.new"), reason="outdated")
'/firefox/new/?reason=outdated'
```

The reason is `urlparams` can handle escaping and URL encoding, and so you don't have to worry about the `?` or `&`s.

And for future knowledge, you can also pass it a `hash` if you need to and it'll put things where they should go, e.g.:
```python
>>> urlparams(reverse("firefox.new"), hash="intro", reason="outdated", version="100.0.0")
'/firefox/new/?reason=outdated&version=100.0.0#intro'
```

Reply:
Thanks for info here!